web-components/src/components/molecules/PrevNextButtons/PrevNextButtons.tsx

```diff
@@ -0,0 +1,44 @@
+import { cn } from '../../../utils/styles/cn';
+import ContainedButton from '../../buttons/ContainedButton';
+import OutlinedButton from '../../buttons/OutlinedButton';
+import ArrowDownIcon from '../../icons/ArrowDownIcon';
+import { useStyles } from './PrevNextButtons.styles';
+
+type Props = {
+  onPrev?: () => void;
+  onSave?: () => void;
```

Comment:
if you look at the code for the `SaveFooter` you'll see the button has `type="submit"` which means that if it's within a `Form` it will just use `onSubmit` from this form on click so no need to provide `onSave` in this case